### ISSUE TITLE
CLI help typo

### ISSUE THREAD

**Issue by bernt-matthias** [KennthShang/PhaBOX#37]
[This](https://github.com/KennthShang/PhaBOX/blob/2b663d52eb8a15f61e9de09847a0273e360ffa0c/src/phabox2/phabox2.py#L205) should probably be `--ccov`?

**Comment by KennthShang** [comment]
Yes, it is `--ccov`. Thanks for catching this. Will update it in the next patch.

Best,
Jiayu

**Comment by bernt-matthias** [comment]
https://github.com/KennthShang/PhaBOX/blob/f90abd7083fdc15976294a9118cfff454416d143/src/phabox2/phabox2.py#L282

```diff
-        portal protein || 84 percent prokaryotic viruses have portal
+        portal         || 84 percent prokaryotic viruses have portal
```

**Comment by bernt-matthias** [comment]
Wonder if the detailed docs for `--task cherry`  miss the description for `--magonly`?

**Comment by bernt-matthias** [comment]
Was also asking myself: The `--aai` argument is used in two tasks. Is it intended that uses can only choose the same value for both tasks?

**Comment by KennthShang** [comment]
Thanks for the corrections. I have revised the mentioned descriptions.

```
Wonder if the detailed docs for --task cherry miss the description for --magonly?
```
Yes, it seems I forgot to add this new option. I have mentioned it to the phabox2.py scripts in the GitHub, and also update the wiki.

```
Was also asking myself: The --aai argument is used in two tasks. Is it intended that uses can only choose the same value for both tasks?
```
Yes, if the user calls the end_to_end task, `--aai`, `--share`, and `--pcov` can only choose the same value. This is because some submodules in CHERRY will reuse the results from PhaGCN. （According to our experiments, the taxonomy classification of virus aids the host prediction).

Thanks for your reports.

Best,
Jiayu

**Comment by KennthShang** [comment]
I will update a new release if we figure out why the tree task is stuck on your data.

By the way, I have updated the WIKI so that users can learn how to install the latest PhaBOX with GitHub. Sorry for the inconvenience. 

Best,
Jiayu

**Comment by KennthShang** [comment]
The above-mentioned issues are fixed in v2.1.5

Best,
Jiayu
